pom.xml

```diff
@@ -190,6 +190,18 @@
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
+      <artifactId>jmh-core</artifactId>
+      <version>1.9</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
+      <artifactId>jmh-generator-annprocess</artifactId>
+      <version>1.9</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Why did you add this dependency?

Reply:
My mistake. I have run a benchmark to check the performance. Thanks for seeing it! :)